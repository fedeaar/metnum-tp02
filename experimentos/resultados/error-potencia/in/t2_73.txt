8.730299000000000000e+06 1.104440000000000000e+05 9.389390000000000000e+05 -3.978560000000000000e+05 2.379974000000000000e+06 -5.467960000000000000e+05 1.509939000000000000e+06 9.652290000000000000e+05 2.781030000000000000e+05 1.568589000000000000e+06 -1.635150000000000000e+05 1.011732000000000000e+06 -1.029659000000000000e+06 2.484674000000000000e+06 6.228860000000000000e+05 9.655000000000000000e+05 1.923098000000000000e+06 -5.395220000000000000e+05 5.425940000000000000e+05 -2.779184000000000000e+06
1.104440000000000000e+05 8.515264000000000000e+06 -1.634208000000000000e+06 -2.005935000000000000e+06 -1.132836000000000000e+06 7.475770000000000000e+05 -4.615130000000000000e+05 1.165698000000000000e+06 -4.204730000000000000e+05 -7.457810000000000000e+05 -1.882938000000000000e+06 -1.280046000000000000e+06 1.272472000000000000e+06 8.871010000000000000e+05 -1.694181000000000000e+06 -8.497550000000000000e+05 8.728540000000000000e+05 3.042973000000000000e+06 2.286470000000000000e+06 1.235726000000000000e+06
9.389390000000000000e+05 -1.634208000000000000e+06 7.820116000000000000e+06 -9.867870000000000000e+05 -1.401270000000000000e+05 -1.192265000000000000e+06 -1.301065000000000000e+06 -4.467500000000000000e+05 9.980860000000000000e+05 -2.790609000000000000e+06 1.822391000000000000e+06 1.589554000000000000e+06 -1.060902000000000000e+06 -1.948970000000000000e+05 -1.029776000000000000e+06 1.476848000000000000e+06 1.814794000000000000e+06 -1.747847000000000000e+06 1.029254000000000000e+06 -2.632644000000000000e+06
-3.978560000000000000e+05 -2.005935000000000000e+06 -9.867870000000000000e+05 6.462639000000000000e+06 8.986700000000000000e+04 1.638318000000000000e+06 1.568523000000000000e+06 -4.927000000000000000e+05 4.623000000000000000e+04 1.212052000000000000e+06 2.414780000000000000e+06 -7.272740000000000000e+05 -8.576370000000000000e+05 -3.620450000000000000e+05 6.753550000000000000e+05 -9.022900000000000000e+04 -2.441064000000000000e+06 -2.078498000000000000e+06 7.380400000000000000e+04 -6.452760000000000000e+05
2.379974000000000000e+06 -1.132836000000000000e+06 -1.401270000000000000e+05 8.986700000000000000e+04 4.981743000000000000e+06 3.160420000000000000e+05 -8.266590000000000000e+05 -2.203940000000000000e+05 1.078220000000000000e+05 3.605410000000000000e+05 1.654908000000000000e+06 2.489880000000000000e+05 -8.004810000000000000e+05 1.203556000000000000e+06 8.386770000000000000e+05 -3.080260000000000000e+05 -1.278827000000000000e+06 -7.191700000000000000e+05 -2.131019000000000000e+06 -6.557870000000000000e+05
-5.467960000000000000e+05 7.475770000000000000e+05 -1.192265000000000000e+06 1.638318000000000000e+06 3.160420000000000000e+05 5.232386000000000000e+06 -3.780290000000000000e+05 -1.548668000000000000e+06 4.894360000000000000e+05 1.313785000000000000e+06 9.295030000000000000e+05 -2.939040000000000000e+06 2.848076000000000000e+06 -3.801720000000000000e+05 2.160776000000000000e+06 7.448540000000000000e+05 -6.902710000000000000e+05 -3.091640000000000000e+05 -1.770718000000000000e+06 -2.651340000000000000e+05
1.509939000000000000e+06 -4.615130000000000000e+05 -1.301065000000000000e+06 1.568523000000000000e+06 -8.266590000000000000e+05 -3.780290000000000000e+05 4.694535000000000000e+06 -9.051100000000000000e+04 5.766600000000000000e+04 1.064523000000000000e+06 1.548371000000000000e+06 2.427250000000000000e+05 -5.880360000000000000e+05 -1.199914000000000000e+06 5.751090000000000000e+05 -2.670960000000000000e+06 -2.848250000000000000e+05 1.680090000000000000e+05 1.449085000000000000e+06 -1.434340000000000000e+05
9.652290000000000000e+05 1.165698000000000000e+06 -4.467500000000000000e+05 -4.927000000000000000e+05 -2.203940000000000000e+05 -1.548668000000000000e+06 -9.051100000000000000e+04 5.829440000000000000e+06 -2.532485000000000000e+06 1.020200000000000000e+05 1.276008000000000000e+06 -2.024618000000000000e+06 -4.235640000000000000e+05 5.082530000000000000e+05 -3.836360000000000000e+05 -3.055940000000000000e+05 9.706070000000000000e+05 2.301410000000000000e+05 2.822173000000000000e+06 -1.427844000000000000e+06
2.781030000000000000e+05 -4.204730000000000000e+05 9.980860000000000000e+05 4.623000000000000000e+04 1.078220000000000000e+05 4.894360000000000000e+05 5.766600000000000000e+04 -2.532485000000000000e+06 4.742919000000000000e+06 5.008900000000000000e+05 3.024850000000000000e+05 1.337914000000000000e+06 -1.398966000000000000e+06 -1.607170000000000000e+05 1.923989000000000000e+06 1.288694000000000000e+06 4.424800000000000000e+04 7.910030000000000000e+05 -8.338560000000000000e+05 -1.367053000000000000e+06
1.568589000000000000e+06 -7.457810000000000000e+05 -2.790609000000000000e+06 1.212052000000000000e+06 3.605410000000000000e+05 1.313785000000000000e+06 1.064523000000000000e+06 1.020200000000000000e+05 5.008900000000000000e+05 4.891797000000000000e+06 2.184606000000000000e+06 -1.656311000000000000e+06 1.536151000000000000e+06 -1.847914000000000000e+06 5.675520000000000000e+05 2.080360000000000000e+05 -1.640226000000000000e+06 -7.493490000000000000e+05 -1.479895000000000000e+06 8.334650000000000000e+05
-1.635150000000000000e+05 -1.882938000000000000e+06 1.822391000000000000e+06 2.414780000000000000e+06 1.654908000000000000e+06 9.295030000000000000e+05 1.548371000000000000e+06 1.276008000000000000e+06 3.024850000000000000e+05 2.184606000000000000e+06 9.586027000000000000e+06 -1.914055000000000000e+06 -1.316637000000000000e+06 -2.354471000000000000e+06 -2.522650000000000000e+05 -1.498933000000000000e+06 -2.182946000000000000e+06 -9.987700000000000000e+05 -1.693240000000000000e+05 -6.151960000000000000e+05
1.011732000000000000e+06 -1.280046000000000000e+06 1.589554000000000000e+06 -7.272740000000000000e+05 2.489880000000000000e+05 -2.939040000000000000e+06 2.427250000000000000e+05 -2.024618000000000000e+06 1.337914000000000000e+06 -1.656311000000000000e+06 -1.914055000000000000e+06 6.046592000000000000e+06 -2.256614000000000000e+06 -4.713400000000000000e+05 4.124120000000000000e+05 -3.542380000000000000e+05 2.289451000000000000e+06 6.653720000000000000e+05 -1.521627000000000000e+06 8.308970000000000000e+05
-1.029659000000000000e+06 1.272472000000000000e+06 -1.060902000000000000e+06 -8.576370000000000000e+05 -8.004810000000000000e+05 2.848076000000000000e+06 -5.880360000000000000e+05 -4.235640000000000000e+05 -1.398966000000000000e+06 1.536151000000000000e+06 -1.316637000000000000e+06 -2.256614000000000000e+06 6.162448000000000000e+06 -1.099870000000000000e+06 8.412100000000000000e+04 8.877800000000000000e+04 -4.478020000000000000e+05 -2.195859000000000000e+06 -1.803070000000000000e+06 2.366307000000000000e+06
2.484674000000000000e+06 8.871010000000000000e+05 -1.948970000000000000e+05 -3.620450000000000000e+05 1.203556000000000000e+06 -3.801720000000000000e+05 -1.199914000000000000e+06 5.082530000000000000e+05 -1.607170000000000000e+05 -1.847914000000000000e+06 -2.354471000000000000e+06 -4.713400000000000000e+05 -1.099870000000000000e+06 8.125730000000000000e+06 -9.583230000000000000e+05 2.937027000000000000e+06 1.564400000000000000e+05 1.558268000000000000e+06 2.087548000000000000e+06 -1.482318000000000000e+06
6.228860000000000000e+05 -1.694181000000000000e+06 -1.029776000000000000e+06 6.753550000000000000e+05 8.386770000000000000e+05 2.160776000000000000e+06 5.751090000000000000e+05 -3.836360000000000000e+05 1.923989000000000000e+06 5.675520000000000000e+05 -2.522650000000000000e+05 4.124120000000000000e+05 8.412100000000000000e+04 -9.583230000000000000e+05 5.288644000000000000e+06 5.646040000000000000e+05 9.107580000000000000e+05 2.950680000000000000e+05 -2.136262000000000000e+06 -1.866555000000000000e+06
9.655000000000000000e+05 -8.497550000000000000e+05 1.476848000000000000e+06 -9.022900000000000000e+04 -3.080260000000000000e+05 7.448540000000000000e+05 -2.670960000000000000e+06 -3.055940000000000000e+05 1.288694000000000000e+06 2.080360000000000000e+05 -1.498933000000000000e+06 -3.542380000000000000e+05 8.877800000000000000e+04 2.937027000000000000e+06 5.646040000000000000e+05 5.737600000000000000e+06 -4.110800000000000000e+05 2.885020000000000000e+05 3.468410000000000000e+05 -2.129748000000000000e+06
1.923098000000000000e+06 8.728540000000000000e+05 1.814794000000000000e+06 -2.441064000000000000e+06 -1.278827000000000000e+06 -6.902710000000000000e+05 -2.848250000000000000e+05 9.706070000000000000e+05 4.424800000000000000e+04 -1.640226000000000000e+06 -2.182946000000000000e+06 2.289451000000000000e+06 -4.478020000000000000e+05 1.564400000000000000e+05 9.107580000000000000e+05 -4.110800000000000000e+05 7.557410000000000000e+06 5.091150000000000000e+05 -1.024490000000000000e+06 -1.644670000000000000e+05
-5.395220000000000000e+05 3.042973000000000000e+06 -1.747847000000000000e+06 -2.078498000000000000e+06 -7.191700000000000000e+05 -3.091640000000000000e+05 1.680090000000000000e+05 2.301410000000000000e+05 7.910030000000000000e+05 -7.493490000000000000e+05 -9.987700000000000000e+05 6.653720000000000000e+05 -2.195859000000000000e+06 1.558268000000000000e+06 2.950680000000000000e+05 2.885020000000000000e+05 5.091150000000000000e+05 6.733737000000000000e+06 2.201209000000000000e+06 -2.760320000000000000e+05
5.425940000000000000e+05 2.286470000000000000e+06 1.029254000000000000e+06 7.380400000000000000e+04 -2.131019000000000000e+06 -1.770718000000000000e+06 1.449085000000000000e+06 2.822173000000000000e+06 -8.338560000000000000e+05 -1.479895000000000000e+06 -1.693240000000000000e+05 -1.521627000000000000e+06 -1.803070000000000000e+06 2.087548000000000000e+06 -2.136262000000000000e+06 3.468410000000000000e+05 -1.024490000000000000e+06 2.201209000000000000e+06 6.966114000000000000e+06 -2.052928000000000000e+06
-2.779184000000000000e+06 1.235726000000000000e+06 -2.632644000000000000e+06 -6.452760000000000000e+05 -6.557870000000000000e+05 -2.651340000000000000e+05 -1.434340000000000000e+05 -1.427844000000000000e+06 -1.367053000000000000e+06 8.334650000000000000e+05 -6.151960000000000000e+05 8.308970000000000000e+05 2.366307000000000000e+06 -1.482318000000000000e+06 -1.866555000000000000e+06 -2.129748000000000000e+06 -1.644670000000000000e+05 -2.760320000000000000e+05 -2.052928000000000000e+06 6.188190000000000000e+06
