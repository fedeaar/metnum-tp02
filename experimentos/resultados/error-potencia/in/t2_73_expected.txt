2.009539769819436222e+07
1.729677213402382284e+07
1.524712557505696826e+07
1.310113938070335798e+07
1.098743056393748708e+07
1.020851790330382623e+07
8.823537714193768799e+06
7.475649836257531308e+06
6.344378015561294742e+06
5.302829052714348771e+06
4.298890539972065017e+06
3.610409048781556543e+06
2.417638672870545648e+06
2.089632341795616085e+06
1.047770360910489690e+06
7.871059196212126408e+05
5.192556096045806771e+05
3.602439881629595766e+05
2.224942734761451138e+05
5.741137085812373698e+04
